pkg/apis/pipeline/v1beta1/taskref_conversion.go

```diff
@@ -10,18 +10,36 @@ func (tr TaskRef) convertTo(ctx context.Context, sink *v1.TaskRef) {
 	sink.Name = tr.Name
 	sink.Kind = v1.TaskKind(tr.Kind)
 	sink.APIVersion = tr.APIVersion
-	// TODO: handle bundle in #4546
 	new := v1.ResolverRef{}
 	tr.ResolverRef.convertTo(ctx, &new)
 	sink.ResolverRef = new
+	tr.convertBundleToResolver(sink)
 }
 
 func (tr *TaskRef) convertFrom(ctx context.Context, source v1.TaskRef) {
 	tr.Name = source.Name
 	tr.Kind = TaskKind(source.Kind)
 	tr.APIVersion = source.APIVersion
-	// TODO: handle bundle in #4546
 	new := ResolverRef{}
 	new.convertFrom(ctx, source.ResolverRef)
 	tr.ResolverRef = new
 }
+
+// convertBundleToResolver converts v1beta1 bundle string to a remote reference with the bundle resolver in v1.
```

Comment:
Can you add a comment about why we are not doing the reverse conversion?

Reply:
Thank you Lee for the comment. Updated.